templates/zimrc

```diff
@@ -1,96 +1,107 @@
-
+################
+# ZIM SETTINGS #
+################
+
+# Set input mode to 'emacs' (default) or 'vi'.
+#zstyle ':zim:input' mode 'vi'
+
+# Select what modules you would like enabled. Modules are sourced in the order given.
+zstyle ':zim' modules \
+    directory environment git git-info history input utility \
+    steeef \
+    zsh-completions completion \
+    zsh-autosuggestions zsh-syntax-highlighting history-substring-search
+
+# Modules setup configuration.
+# See https://github.com/zimfw/zimfw/blob/develop/README.md#module-customization
+zstyle ':zim:module' zsh-completions 'url' 'zsh-users/zsh-completions'
+zstyle ':zim:module' zsh-autosuggestions 'url' 'zsh-users/zsh-autosuggestions'
+zstyle ':zim:module' zsh-syntax-highlighting 'url' 'zsh-users/zsh-syntax-highlighting'
 
 ###################
 # MODULE SETTINGS #
 ###################
 
 #
-# Prompt
+# completion
 #
 
-# Set your desired prompt here
-zprompt_theme='steeef'
+# Set a custom path for the completion dump file.
+# If none is provided, the default ${ZDOTDIR:-${HOME}}/.zcompdump is used.
+#zstyle ':zim:completion' dumpfile "${ZDOTDIR:-${HOME}}/.zcompdump-${ZSH_VERSION}"
 
 #
-# Completion
+# environment
 #
 
-# Set an optional host-specific filename for the completion cache file. If none
-# is provided, the default '.zcompdump' is used.
-#zcompdump_file=".zcompdump-${HOST}-${ZSH_VERSION}"
+# Set the string below to the desired terminal title format string.
+# The terminal title is redrawn upon directory change, however, variables like
+# ${PWD} are only evaluated once. Use prompt expansion strings for dynamic data.
+# See http://zsh.sourceforge.net/Doc/Release/Prompt-Expansion.html#Simple-Prompt-Escapes
+# For example, '%n@%m: %1~' corresponds to 'username@host: /current/directory'.
+#zstyle ':zim:environment' termtitle '%n@%m: %1~'
 
 #
-# Utility
+# history
 #
 
-# Uncomment to enable spelling correction prompt for commands. See:
-# http://zsh.sourceforge.net/Doc/Release/Options.html#Input_002fOutput
-#setopt CORRECT
-
-# Set custom spelling correction prompt
-#SPROMPT='zsh: correct %F{red}%R%f to %F{green}%r%f [nyae]? '
+# Save the history in a custom file path.
+# If none is provided, the default ${ZDOTDIR:-${HOME}}/.zhistory is used.
+#HISTFILE=${ZDOTDIR:-${HOME}}/.zsh_history
 
 #
-# Environment
+# input
 #
 
-# Set the string below to the desired terminal title format string.
-# The terminal title is redrawn upon directory change, however, variables like 
-# ${PWD} are only evaluated once. Use prompt expansion strings for dynamic data:
-#   http://zsh.sourceforge.net/Doc/Release/Prompt-Expansion.html#Simple-Prompt-Escapes
-# The example below uses the following format: 'username@host:/current/directory'
-ztermtitle='%n@%m:%~'
+# Enable double-dot expansion.
+# This appends '../' to your input for each '.' you type after an initial '..'
+#zstyle ':zim:input' double-dot-expand yes
 
 #
-# Input
+# pacman
```

Comment:
_pacman_ and _ssh_ sections are 'dangling' - these two modules are not in the default module selection